upgrade/1.0.11/Stage_2.md

```diff
@@ -0,0 +1,126 @@
+# Stage 2 - Kubernetes Node Image Upgrade
+
+> NOTE: During the CSM-1.0.1 install the LiveCD containing the initial install files for this system should have been unmounted from the master node when rebooting into the Kubernetes cluster. The scripts run in this section will also attempt to unmount/eject it if found to ensure the USB stick does not get erased.
+
+>**`IMPORTANT:`**
+> 
+> Reminder: Before running any upgrade scripts, be sure the Cray CLI output format is reset to default by running the following command:
+>
+>```bash
+> ncn# unset CRAY_FORMAT
+>```
+
+## Stage 2.1
+
+1. Run `ncn-upgrade-k8s-master.sh` for `ncn-m002`. Follow output of the script carefully. The script will pause for manual interaction.
+
+    ```bash
+    ncn-m001# /usr/share/doc/csm/upgrade/1.0.11/scripts/upgrade/ncn-upgrade-k8s-master.sh ncn-m002
+    ```
+    
+    > NOTE: You may need to reset the root password for each node after it is rebooted
+
+1. Repeat the previous step for each other master node **excluding `ncn-m001`**, one at a time.
+
+## Stage 2.2
+
+1. Run `ncn-upgrade-k8s-worker.sh` for `ncn-w001`. Follow output of the script carefully. The script will pause for manual interaction.
+
+    > NOTE: It is expected that some pods may be in bad states during a worker node upgrade. This is due to a temporary lack of computing resources. Once the worker node has been upgraded and has rejoined the cluster, those pods will be up and running again. All critical services have more than one replica so that if one pod is down, the service is still available.
+
+    > NOTE: It is possible that some postgres clusters may report errors when `ncn-upgrade-k8s-worker.sh` runs and checks the postgres clusters. If errors are reported indicating that a cluster does not have a leader or is lagging, it is recommended to re-run the `ncn-upgrade-k8s-worker.sh` again after waiting about 30 minutes to give the clusters time to resume to a healthy state. If postgres clusters still report leader or lag issues, then refer to [Troubleshoot Postgres Database](../../operations/kubernetes/Troubleshoot_Postgres_Database.md).
```

Comment:
"postgres" should be referred to as "PostgreSQL" ( see https://www.postgresql.org/ )